database/callbacks_disable/beforeMigrate__disable_subscription.sql

```diff
@@ -0,0 +1,8 @@
+DO $$
+DECLARE
+  _sub TEXT := '${env}_${network}_alltables';
+BEGIN
+  IF EXISTS (SELECT 1 FROM pg_subscription WHERE subname = _sub) THEN
+     EXECUTE format('ALTER SUBSCRIPTION %I DISABLE;', _sub);
+  END IF;
```

Comment:
Would it be helpful to log both branches, the` if` and the `else`, for better visibility/debuggability?

Something like: 
```
RAISE NOTICE 'Subscription % does not exist. Skipping.', _sub;
```